### DISCUSSION TITLE
SCript not running

### DISCUSSION THREAD

**Discussion by pquodling** [tteck/Proxmox#3880] [Help Needed]
I have a SYnology DS918 and am trying to get PBS running per the instructions at https://www.derekseaman.com/2023/04/how-to-setup-synology-nfs-for-proxmox-backup-server-datastore.html

I get to the section where it says to get the pbs-post-install script i.e. the following command. 

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/post-pbs-install.sh)"

I enter that, hit return, and it returns to a prompt - doesn't appear to execute or even download - doesn't return any error. 

I can ping github so the network is there...

**Comment by tteck** [comment]
🌐 It seems that you're facing a network problem, _probably_ linked to DNS.

**Comment by MickLesk** [comment]
Whats your output when you run: 
wget -d - https://github.com/tteck/Proxmox/raw/main/misc/post-pbs-install.sh ?

Thats the debug of wget and save the file locally. If nothing happens, it will probably be a DNS or internet problem as tteck says.

**Comment by pquodling** [comment]
Bingo - it was DNS - I had the DNS pointing to my router - I changed to 1.1.1.1 with a secondary of 8.8.8.8 and it appears to be working just fine now. 

Thanks all
